packages/bsync/src/db/schema/subs_op.ts

```diff
@@ -0,0 +1,17 @@
+import { ColumnType, GeneratedAlways, Selectable } from 'kysely'
+
+export interface SubsOp {
+  id: GeneratedAlways<number>
+  actorDid: string
+  // https://github.com/kysely-org/kysely/issues/137
+  entitlements: ColumnType<string[], string, string>
+  createdAt: GeneratedAlways<Date>
+}
+
+export type SubsOpEntry = Selectable<SubsOp>
+
+export const tableName = 'subs_op'
+
+export type PartialDB = { [tableName]: SubsOp }
+
+export const createSubsOpChannel = 'subs_op_create' // used with listen/notify
```

Comment:
We may want to wire this up here as well: https://github.com/bluesky-social/atproto/blob/d634f9d6d858dc26c69443e9c2e1f1e20ed381a3/packages/bsync/src/index.ts#L102-L109